### ISSUE TITLE
Last ACF Update broke standard fields automatic conversion

### ISSUE THREAD

**Issue by JulianCataldo13** [qtranslate/qtranslate-xt#988]
Hello there.

It seems like the latest ACF, while "convert standard fields" is checked in ACF-Qtranslate settings, does not bind anymore to standard Qtranslate functions.

Workaround : convert all translatable fields manually.

**Comment by MK-RD** [comment]
MF 87.0 (64-Bit)
WP 5.7
QTX 3.9.3
ACF 5.9.5 (PRO)

I know there is follow-up problem with the acf fields in connection with the browser cache, which can be solved via a hack.
But I cannot confirm the behavior you describe!

So in what environment was this behavior observed?

**Comment by herrvigg** [comment]
I'm testing with ACF 5.9.5 (Free) and I don't see any particular problem.
Can you try to debug this? You should see these scripts to be loaded:

```
/wp-content/plugins/qtranslate-xt/modules/acf/assets/acf_5/main.js
/wp-content/plugins/qtranslate-xt/modules/acf/assets/common.js
/wp-content/plugins/qtranslate-xt/modules/acf/assets/acf_5/qtranslatex.js
```

**Comment by herrvigg** [comment]
@JulianCataldo13 I found a potential problem with async loading in the ACF module, similar to #946 (ready vs load). I just pushed a patch to master, maybe that solves your problem.

**Comment by herrvigg** [comment]
With the last version on master, ACF is now bundled with webpack so you will only have a single script:
```
/wp-content/plugins/qtranslate-xt/dist/modules/acf.js
```

**Comment by herrvigg** [comment]
@JulianCataldo13 please could you try with the current version on this `master` branch (just download the last zip)?

**Comment by sermalefico** [comment]
i think this no works. The automatic conversion don´t work

**Comment by herrvigg** [comment]
Should now be fixed in master, works with ACF 5.12.4 and ACF 6, see #1255.
